config/tf_modules/gcp-mysql/outputs.tf

```diff
@@ -0,0 +1,16 @@
+output "db_user" {
```

Comment:
Add a comment here that this list of outputs needs to be the same as the ones for gcp-postgres and should match the ones at `gcp_k8s_service.py:handle_rds_link()`

Reply:
done!